library/core/src/main/java/com/google/android/exoplayer2/audio/SonicAudioProcessor.java

```diff
@@ -66,6 +66,7 @@
   private int sampleRateHz;
   private float speed;
   private float pitch;
+  private long videoOffsetUs;
```

Comment:
The changes to this class don't seem to do anything (in particular `videoOffsetUs` doesn't seem to be read by anything? Ditto to the change in `DefaultAudioSink`.

Reply:
@ojw28  this is used on this class:
https://github.com/google/ExoPlayer/pull/5766/files#diff-d7c4edd055ad44ffd9ee2b757ac41786

when we set the playbackParameters e.g:
```suggestion
  final long videoOffsetUs = 500 * 1000; // video offset in micro seconds
      PlaybackParameters playbackParameters = new PlaybackParameters(1f, 1f, videoOffsetUs, false);
      player.setPlaybackParameters(playbackParameters);
```

somehow exoplayer will copying that parameters into SonicAudioProcessor:
library/core/src/main/java/com/google/android/exoplayer2/audio/DefaultAudioSink.java

hence I am adding the video offset on this class too.